app/views/layouts/application.html.erb

```diff
@@ -12,9 +12,12 @@
   </head>
 
   <body>
-    <%= render 'shared/navbar' %>
-    <%= render 'shared/flashes' %>
-
-    <%= yield %>
+    <div class="row justify-content-center">
+      <div class="col-4">
```

Comment:
Wouldn't this lead to the content taking up 1/3rd of the available space (i.e., 4 of 12 parts)?

Reply:
yes, I agree; that was Zach´s suggestion... The point is just to 1/ create a mobile view just in 1 place (here) so to avoid differences between the different views 2/ agree together/with another teacher on which mobile view we want to render